src/generators/markdowndocgenerator/itemrenderer.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import { MarkdownDocGeneratorOptionsManager } from './optionsmanager';
+
+import { INotebookHeading } from '../shared';
+
+import * as React from 'react';
+
+export function markdownDocItemRenderer(
+  options: MarkdownDocGeneratorOptionsManager,
+  item: INotebookHeading
```

Comment:
Since the markdown document also uses the `INotebookHeading`, perhaps we should come up with another name for it.